services/invitations/README.md

```diff
@@ -7,11 +7,2789 @@ Users invited via this Invitation Manager (libre graph API) will have `userType=
 The corresponding CS3 API [user types](https://cs3org.github.io/cs3apis/#cs3.identity.user.v1beta1.UserType) used to reperesent this are: `USER_TYPE_GUEST` and `USER_TYPE_PRIMARY`.
 
 
+## Subcommand
+
+This service introduces the `invitations` subcommand to `ocis`.
```

Comment:
Q: with subcommand you mean a CLI? In case we need to rephrase the stuff a bit and propose an example, see: https://doc.owncloud.com/ocis/next/deployment/services/s-list/storage-users.html#manage-unfinished-uploads